docs/dcomibmtoolsattachtimeout.md

```diff
@@ -32,14 +32,17 @@ Specify a time that an application should wait when attempting to connect to a t
 
 | Setting | Value                          | Default                          |
 |---------|--------------------------------|----------------------------------|
-| `<ms>`  | [1 millisecond or greater]     | 120000 milliseconds (120 seconds)|
+| `<ms>`  | [500 milliseconds or greater]     | 120000 milliseconds (120 seconds)|
+
+If you specify a value less than 500 milliseconds, no connection attempt is made.
```

Comment:
Actually the value has to be greater than `500` milliseconds, no connection attempt will be made if it is less or equal than that.
Here is the related code snippet [1]:
```
MAXIMUM_ATTACH_TIMEOUT = Integer.getInteger("com.ibm.tools.attach.timeout", DEFAULT_ATTACH_TIMEOUT).intValue(); //$NON-NLS-1$

int timeout = 500; /* start small in case there is a rogue process which is eating semaphores, grow big in case of system load. */
while (timeout < MAXIMUM_ATTACH_TIMEOUT) {
    ...
    tryAttachTarget(timeout);
    ...
}
```
Apology for not making this clear in the offline conversation.

[1] https://github.com/eclipse/openj9/blob/18c50361e42d18a16ed815e9ea0731853cee9c95/jcl/src/jdk.attach/share/classes/com/ibm/tools/attach/attacher/OpenJ9VirtualMachine.java#L155-L168

Reply:
No worries, fix made. Thanks